org.gpodder.gpodder.json

```diff
@@ -1,7 +1,7 @@
 {
   "app-id": "org.gpodder.gpodder",
   "runtime": "org.gnome.Platform",
-  "runtime-version": "45",
+  "runtime-version": "46",
```

Comment:
Any reason not to upgrade to version 47?

Reply:
There was a few errors.